jihyeon/middleware/auth.js

```diff
@@ -0,0 +1,16 @@
+const jwt = require("jsonwebtoken");
+
+const validateToken = async (req, res, next) => {
+  try {
+    const token = req.headers.authorization;
+    if (!token) return res.status(403).json("ACCESS_DENIED");
+
+    const decoded = jwt.verify(token, process.env.SECRETKEY);
+    req.user = decoded.user_id;
+    next();
```

Comment:
1. status code 403은 인증은 되었지만, 권한이 없는 상태입니다. 지금의 상황은 인증 자체가 안 되었기 때문에 401이 더 적절한 status code 입니다.
```suggestion
  try {
    const token = req.headers.authorization;
    
    if (!token) return res.status(401).json("ACCESS_DENIED");

    const decoded = jwt.verify(token, process.env.SECRETKEY);
    req.user = decoded.user_id;
   
     next();
```